crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -995,15 +1003,15 @@ functions are instances of that class:
 ```py
 def f(): ...
 
-reveal_type(f.__defaults__)  # revealed: @Todo(full tuple[...] support) | None
-reveal_type(f.__kwdefaults__)  # revealed: @Todo(generics) | None
+reveal_type(f.__defaults__)  # revealed: @Todo(full tuple[...] support)
+reveal_type(f.__kwdefaults__)  # revealed: @Todo(generics)
```

Comment:
So this is a little weird, but I'm not it's important enough to fix for this test specifically. A static member lookup of `getattr_static(f, "__defaults__")` still gives us `@Todo(full tuple[...] support) | None`, but when we look up `__get__` on this type, we get `@Todo(…)` from the first element of the union, and `Symbol::Unbound` from the second element of the union (`None`). We then combine those into a possibly-unbound `@Todo(…)` type, which is why the `None` has vanished from the union.